src/main/java/com/flywithingryd/IngrydAirways/model/Itinerary.java

```diff
@@ -0,0 +1,19 @@
+package com.flywithingryd.IngrydAirways.model;
+
+import jakarta.persistence.*;
+
+import java.util.Date;
+
+@Entity(name = "itinerary")
+public class Itinerary {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+    private Passenger passenger;
+    private Reservation reservation;
+
+    @ManyToOne
+    @JoinColumn()
+    private Flight flight;
```

Comment:
I thought we agreed to remove flight from itinerary as reservation already holds it

Reply:
Same for this.. the description u gave on Whatsapp yesterday was the reason for adding.. but u see i left the joinToColumn empty.. where is this linking to on flight table

Reply:
My apologies but we concluded on the final implementation on call so i thought you took them down, plus joincolumn is usually added in the inverse side of a n entity relationship